ecl/eclcc/eclcc.hpp

```diff
@@ -139,6 +139,7 @@ const char * const helpText[] = {
     "?!  -fcheckAsserts          Check ASSERT() statements",
     "?!  -fcheckDuplicateThreshold=n Warn if SEQUENTIAL or workflow may duplicate more than n%% activities",
     "?!  -fexportDependencies    Generate information about inter-definition dependencies",
+    "!   -fgenerateIR            Output the internal representation for the query to stdout"
```

Comment:
Should generateIRAfterTransform and irOptions be included in the help output?

Reply:
Bug: missing coma at the end of line 142 causing the 142 and 143 to be concatenated together.

Reply:
I didn't include generateIRAfterTransform and irOptions because they are a bit too obscure.  We should really have documentation on all options somewhere else though.